sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/sqlCompiler/compiler/DBSPCompiler.java

```diff
@@ -355,6 +358,59 @@ void printMessages(@Nullable CreateViewStatement statement) {
         }
     }
 
+    void validateViewProperty(SqlFragment key, SqlFragment value) {
+        CalciteObject node = CalciteObject.create(key.getParserPosition());
+        String keyString = key.getString();
+        //noinspection SwitchStatementWithTooFewBranches
+        switch (keyString) {
+            case "connectors":
+                this.validateConnectorsProperty(node, key, value);
+                break;
+            default:
+                throw new CompilationError("Unknown property " + Utilities.singleQuote(keyString), node);
+        }
+    }
+
+    void validateBooleanProperty(CalciteObject node, SqlFragment key, SqlFragment value) {
+        String vs = value.getString();
+        if (vs.equalsIgnoreCase("true") || vs.equalsIgnoreCase("false"))
+            return;
+        throw new CompilationError("Expected a boolean value for property " +
+                Utilities.singleQuote(key.getString()), node);
+    }
+
+    void validateConnectorsProperty(CalciteObject node, SqlFragment key, SqlFragment value) {
```

Comment:
This is probably redundant. The `connectors` attribute is already validated in the manager. No need to check the same things twice.

Reply:
this compiler gives much faster feedback

Reply:
This happens before rust compilation so feedback should be fast